src/app/components/MessageBanner/index.tsx

```diff
@@ -12,12 +12,12 @@ import styles from './index.styles';
 import { ServiceContext } from '../../contexts/ServiceContext';
 
 interface MessageBannerProps {
-  summaries: Summary[];
+  summary: Summary;
   title: string;
   position: number;
 }
 
-const MessageBanner = ({ summaries, title, position }: MessageBannerProps) => {
+const MessageBanner = ({ summary, title, position }: MessageBannerProps) => {
```

Comment:
We also need to update the docs! https://github.com/bbc/simorgh/blob/latest/src/app/components/MessageBanner/README.md#messagebanner